src/widget/inline.svelte

```diff
@@ -0,0 +1,230 @@
+<script context="module" lang="ts">
+  import { Config, FRUser, type GetTokensOptions, SessionManager, TokenManager, UserManager } from '@forgerock/javascript-sdk';
+  import type { StepOptions } from '@forgerock/javascript-sdk/lib/auth/interfaces';
+  import { get } from 'svelte/store';
+
+  // Import store types
+  import type { JourneyStore, JourneyStoreValue } from '$journey/journey.store';
+  import type { OAuthStore, OAuthTokenStoreValue } from '$lib/oauth/oauth.store';
+  import type { UserStore, UserStoreValue } from '$lib/user/user.store';
+
+  import './main.css';
+
+  export interface Response {
+    journey?: JourneyStoreValue;
+    oauth?: OAuthTokenStoreValue;
+    user?: UserStoreValue;
+  }
+  export interface JourneyOptions {
+    config?: StepOptions;
+    oauth?: boolean; // defaults to true
+    user?: boolean; // defaults to true
+  }
+
+  let callMounted: (form: HTMLFormElement) => void;
+  let journeyStore: JourneyStore;
+  let oauthStore: OAuthStore;
+  let returnError: (response: Response) => void;
+  let returnResponse: (response: Response) => void;
+  let userStore: UserStore;
+
+  export const journey = {
```

Comment:
This portion of the component kind of blew up in size. So much of this is duplicated between the modal and inline component, but I'm ensure how to disentangle them, atm. Will do it in a future story.